apps/backoffice-v2/src/domains/alerts/fetchers.ts

```diff
@@ -46,6 +46,52 @@ export const AlertTypes = [
   AlertType.UNUSUAL_PATTERN,
 ] as const satisfies ReadonlyArray<TObjectValues<typeof AlertType>>;
 
+export const AlertState = {
+  TRIGGERED: 'Triggered',
+  UNDER_REVIEW: 'UnderReview',
+  ESCALATED: 'Escalated',
+  RESOLVED: 'Resolved',
+  ACKNOWLEDGED: 'Acknowledged',
+  DISMISSED: 'Dismissed',
+  REJECTED: 'Rejected',
+  NOT_SUSPICIOUS: 'NotSuspicious',
+} as const;
+
+export const AlertStates = [
+  AlertState.TRIGGERED,
+  AlertState.UNDER_REVIEW,
+  AlertState.ESCALATED,
+  AlertState.RESOLVED,
+  AlertState.ACKNOWLEDGED,
+  AlertState.DISMISSED,
+  AlertState.REJECTED,
+  AlertState.NOT_SUSPICIOUS,
+] as const satisfies ReadonlyArray<TObjectValues<typeof AlertState>>;
+
+export const alertStateToDecision = {
+  REJECTED: 'Reject',
+  NOT_SUSPICIOUS: 'Not Suspicious',
+  REVERT_DECISION: 'Revert Decision',
+} as const satisfies Partial<Record<keyof typeof AlertState | (string & {}), string>>;
+
+export const alertDecisionToState = {
+  REJECT: 'Rejected',
+  NOT_SUSPICIOUS: 'NotSuspicious',
+  REVERT_DECISION: 'Triggered',
+} as const satisfies Partial<Record<keyof typeof AlertState | (string & {}), string>>;
+
+export type TAlertSeverity = (typeof AlertSeverities)[number];
+
+export type TAlertSeverities = typeof AlertSeverities;
+
+export type TAlertType = (typeof AlertTypes)[number];
+
+export type TAlertTypes = typeof AlertTypes;
+
+export type TAlertState = (typeof AlertStates)[number];
+
+export type TAlertStates = typeof AlertStates;
```

Comment:
Until the day we can share Prisma enums,  let's discuss where these enums should sit.